src/fvm_lowered_cell_impl.hpp

```diff
@@ -203,11 +203,9 @@ fvm_integration_result fvm_lowered_cell_impl<Backend>::integrate(
 
         // Integrate mechanism state.
 
-        PE(advance_integrate_state);
```

Comment:
We don't want to measure this any more?

Reply:
Ah, I see, this is supposed to come out of modcc now

Reply:
Yes. We only mark "leaf" nodes in the profiling tree, and the leaf nodes are now inside the calls to `nrn_current` and `nrn_state`